ccm_web/client/src/components/FeatureCard.tsx

```diff
@@ -24,27 +24,29 @@ function FeatureCard (props: FeatureUIProps): JSX.Element {
   const classes = useStyles()
 
   return (
-    <Link className={classes.cardLink} to={props.route} >
-      <Card className={`${classes.root}`} variant="outlined" tabIndex={props.data.ordinality}>
-        <CardContent>
-          <Grid container>
-            <Grid item xs={12}>
-              <div className={classes.centered}>
-                {props.icon}
-              </div>
-              <div className={classes.centered}>
-                <Typography className={classes.title} color="textPrimary" gutterBottom>
-                  {props.data.title}
-                </Typography>
-                <Typography variant="body2" component="p" color="textSecondary">
-                  {props.data.description}
-                </Typography>
-              </div>
+    <RouterLink className={classes.cardLink} to={props.route} tabIndex={-1}>
+      <Card variant='outlined'>
+        <CardActionArea>
```

Comment:
Replaced `tabIndex={props.data.ordinality}` on `Card` with `CardActionArea` to add the card to the sequential tab order and add interaction styling. See https://mui.com/components/cards/#primary-action